docs/src/components/Navigation.tsx

```diff
@@ -4,72 +4,65 @@ import { Text } from '@marigold/components';
 
 export const Navigation: React.FC = () => {
   return (
-    <div>
-      <Link to="/start">
-        <Text>Home</Text>
-      </Link>
+    <Text>
+      <Link to="/start">Marigold</Link>
       <ul>
         <li>
-          <Text>Design Language</Text>
+          <b>Guides</b>
+          <ul>
+            <li>Installation</li>
+            <li>Theme & Variants</li>
+          </ul>
+        </li>
+        <li>
+          <b>Foundation</b>
           <ul>
             <li>
-              <Text>Brand</Text>
-            </li>
-            <li>
-              <Text>Logos</Text>
+              <Link to="/foundation/theming">Theming</Link>
             </li>
+            <li>Layout</li>
+            <li>Box Primitive</li>
             <li>
-              <Text>Guidelines</Text>
+              <Link to="/foundation/icons">Iconography</Link>
             </li>
+            {/* <li>
```

Comment:
We should make the colors depending on a theme, right?
I think it's better to delete instead of comment until we have it in the right way. Do you agree?

Reply:
Yes I didn't want to delete it so that we don't forget that there already exists a nice page about colors, but yes it should not appear in the menu like this